sql/path.h

```diff
@@ -599,11 +599,7 @@ struct Path {
         }
     }
 
-    Path(Path && other) noexcept
```

Comment:
Don't do this.  The function is inlined on purpose so that the construct/move/destroy cycle can be inlined by the compiler.